### ISSUE TITLE
Add latest patch analysis

### ISSUE THREAD

**Issue by TimJentzsch** [zachkont/dotaUpdatesBot#27]
Add a command (`/dotaupdate` or similar) that displays the latest patch analysis from /u/SirBelvedere or /u/Magesunite

**Comment by zachkont** [comment]
I don't see a lot of value for that so I will be skipping this. However, I will leave this open on case someone else wants to implement it

**Comment by TimJentzsch** [comment]
True, it's kinda redundant with the Dota2Feeder channel, but, well, maybe someone still wants to make it.

**Comment by Rohithgilla12** [comment]
Hey,I can do this thing :+1:

**Comment by zachkont** [comment]
Go for it @Rohithgilla12

**Comment by Rohithgilla12** [comment]
Awesome, is there any restrictions on website to be used or can I use any website

**Comment by zachkont** [comment]
I'm not sure what you've got in mind. The original issue opened by Tim assumes using the already saved reddit post. We are open to suggestions though, what do you have in mind?

**Comment by Rohithgilla12** [comment]
My idea was, getting patch analysis from Dota bluff
Which is the top website for patch analysis

**Comment by zachkont** [comment]
Sure, that could be nice
